aws-api/src/main/java/com/amplifyframework/api/aws/SubscriptionOperation.java

```diff
@@ -41,6 +43,7 @@
 
     private String subscriptionId;
```

Comment:
Can this be `final` now that you assign it in the operation constructor?

Reply:
Unfortunately no because I can't assign it until the subscription actually starts.